libbeat/docs/monitoring/configuring.asciidoc

```diff
@@ -0,0 +1,178 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[role="xpack"]
+[[configuring-monitoring]]
+== Configuring monitoring for {beatname_uc}
+++++
+<titleabbrev>Configure monitoring</titleabbrev>
+++++
+
+{monitoring} enables you to easily monitor {beatname_uc} from {kib}. For more
+information, see {kibana-ref}/xpack-monitoring.html[Monitoring].
+
+. Configure {beatname_uc} to send metrics to your {es} cluster by setting the
+`xpack.monitoring` settings in the {beatname_uc} configuration file. For
+example:
++
+--
+[source, yml]
+--------------------
+xpack.monitoring:
+  enabled: true
+  elasticsearch:
+    url: ["https://example.com:9200", "https://example2.com:9200"]
```

Comment:
setting is `hosts`, not `url`.

The reference config contains all settings with defaults + docs: https://github.com/elastic/beats/blob/master/libbeat/_meta/config.reference.yml#L787

Reply:
Just notices the reference config is missing period and buffer_size. Can you update these in the config.reference.yml file as well?

To some extend we try to keep the reference config file in sync with the docs.